### DISCUSSION TITLE
Change track names to Audio or Midi instead of Column

### DISCUSSION THREAD

**Discussion by sebmu7** [helgoboss/helgobox#1096] [Ideas]
I think it would be less confusing if the track names refers to the chosen input instead of the generic name "Column".

**Comment by helgoboss** [comment]
How would you number them?

Important to know in this context: There's actually no such thing as a MIDI or audio column. All columns support both, exactly as in REAPER. The only thing that is tied to MIDI or audio is the recording input.

**Comment by sebmu7** [comment]
Thank you for your answer. My problem is when I start a new project with playtime, I always use a mix of audio and midi for the template. At my first attempts, I didn't know which track was with midi or audio input. Now I know it is 4 tracks audio and 4 tracks midi after I looked at each track separately. My idea was to name the tracks Audio 1, Audio 2, etc. and Midi 1, Midi 2 etc., but I agree that it's not following the philosophy of Reaper. Maybe adding an icon, a color or a visual cue to let the user know at a glance which track has an input in midi or audio?

**Comment by helgoboss** [comment]
I think the problem might be that the "Input" section of the column track at the bottom is not always visible (if it's not high enough). And even if it's visible, the small icon to the left of the input (the audio or MIDI jack) is very small.

Maybe adding a slightly bigger icon directly to the column header?

**Reply by sebmu7** [reply]
I didn't realize there was already icons at the bottom!! Adding a slightly bigger icon to the column header sounds fantastic since the mixer is not always visible.

**Comment by helgoboss** [comment]
Will be implemented as issue #1104.
